plugin/etcd/README.md

```diff
@@ -107,6 +113,7 @@ etcd skydns.local {
 ...
 ~~~
 
+Before getting started with examples, please setup `etcdctl` (with etcdv3 API) as explained [here](https://coreos.com/etcd/docs/latest/dev-guide/interacting_v3.html). This will help you put sample keys to you etcd server easily.
```

Comment:
Mention that `curl <something>` doesn't work anymore with etcd3?

Reply:
I updated the paragraph, PTAL